VRDR/DeathRecord.cs

```diff
@@ -7270,9 +7270,14 @@ public string InjuryDescription
         {
             get
             {
-                if (InjuryIncidentObs?.Value != null)
+                CodeableConcept concept = InjuryIncidentObs?.Value as CodeableConcept;
+                if(concept != null)
                 {
-                    return Convert.ToString(InjuryIncidentObs.Value);
+                    Dictionary<string, string> dict = CodeableConceptToDict(concept);
```

Comment:
Why not just return concept.Text instead of going through the trouble of converting it to a dictionary? The one difference I see is that converting causes null to be returned in the case of an empty string... doesn't seem an important distinction here?

Reply:
Good simplification.